server/.vscode/settings.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
Is this file created by default while setting up the boilerplate .NET API? If not, can you please remove the .vscode folder and add .vscode to the gitignore?

Reply:
Removed it from git